apps/server/src/modules/authentication/services/authentication.service.ts

```diff
@@ -33,6 +34,9 @@ export class AuthenticationService {
 		if (!account) {
 			throw new UnauthorizedLoggableException(username, systemId);
 		}
+		if (account.deactivatedAt != null && account.deactivatedAt.getDate() <= Date.now()) {
```

Comment:
Shouldn't it be
```suggestion
		if (account.deactivatedAt != null && account.deactivatedAt.getTime() <= Date.now().getTime()) {
```
?